cap/modules/records/serializers/schemas/json.py

```diff
@@ -122,6 +122,10 @@ def get_access(self, obj):
                 for index, role_id in enumerate(permission['roles']):
                     permission['roles'][index] = get_role_name_by_id(role_id)
 
+        for key in access.keys():
+            new_key = key.replace('deposit-', '')
+            access[new_key] = access.pop(key)
```

Comment:
you can iterate just once (maybe bit more readable also)
```suggestion
     access = {
            k.replace('deposit-', ''): {
                'users': [
                    get_user_email_by_id(user_id) for user_id in v['users']
                ],
                'roles': [
                    get_role_name_by_id(role_id) for role_id in v['roles']
                ]
            }
            for k, v in obj['metadata']['_access'].items()
        }

```